servers/visual/visual_server_scene.cpp

```diff
@@ -52,6 +52,21 @@ void VisualServerScene::camera_set_perspective(RID p_camera, float p_fovy_degree
 	camera->zfar = p_z_far;
 }
 
+void VisualServerScene::camera_set_oblique(RID p_camera, float p_fovy_degrees, Transform p_camera_gt, Vector3 p_ob_normal, Vector3 p_ob_position, float p_ob_offset, float p_z_near, float p_z_far) {
```

Comment:
Could be `const Transform &p_camera_gt` if it's read-only. The same goes for `Vector3` parameters.

Also applies to other similar occurrence.

Reply:
omw